src/lessons/Data-Structures/01.md

```diff
@@ -0,0 +1,33 @@
+## Welcome to ProtoSchool!
+
+Before diving into code challenges, let's take a minute to look at a few of the
+concepts that underly the decentralized web. Unlike our other tutorials, this one is
+code-free and designed especially to introduce you to some of the key terms you'll
+encounter throughout ProtoSchool.
+
+Let's get started!
+
+
+## Data structures
+
+Whether you're a programmer or not, you're surrounded by data structures every day. Lists, dictionaries, and catalogs all help us organize information and take into account the relationships between various pieces of data.
+
+From [Wikipedia](https://en.wikipedia.org/wiki/Data_structure):
+
+> In computer science, a data structure is a data organization, management and storage format that enables efficient access and modification.[1][2][3] More precisely, a data structure is a collection of data values, the relationships among them, and the functions or operations that can be applied to the data.
+
+In programming, data structures are everywhere. The way you organize data into variables in order to use them in programs involves any from dozens to millions of data structures. If you're a developer, you're probably familiar with common data structures like arrays, objects, graphs, etc.
+
+## Decentralized data structures
+
+On the decentralized web, where we access data directly from our peers rather than
+from a central authority, we need specialized data structures that allow us to verify
+and link between various pieces of content.
+
+Data structures that are using decentralized systems need to be <strong>verifiable</strong>. On a single
```

Comment:
You should be able to use markdown syntax here instead of html <strong>. Some markdown parsers are not forgiving about *any* HTML, even safe html like this.